security/authorization/util/util_test.go

```diff
@@ -0,0 +1,123 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package engine
+
+import (
+	"testing"
+
+	expr "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
+	"google.golang.org/grpc/internal/grpctest"
+
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/checker/decls"
+)
+
+type s struct {
+	grpctest.Tester
+}
+
+func Test(t *testing.T) {
+	grpctest.RunSubTests(t, s{})
+}
+
+func (s) TestStringConvert(t *testing.T) {
+	declarations := []*expr.Decl{
+		decls.NewIdent("request.url_path", decls.String, nil),
+		decls.NewIdent("request.host", decls.String, nil),
+		decls.NewIdent("connection.uri_san_peer_certificate", decls.String, nil)}
+	env, err := cel.NewEnv()
+	if err != nil {
+		t.Fatalf("Failed to create the CEL environment")
+	}
+	for _, test := range []struct {
+		desc             string
+		wantEvalOutcome  bool
+		wantParsingError bool
+		wantEvalError    bool
+		expr             string
+		authzArgs        map[string]interface{}
+	}{
+		{
+			desc:            "Test 1 single primitive match",
```

Comment:
As I mentioned in the other PR, there is no need to index these test names. Just "single primitive match" should be good enough.